packages/react-instantsearch-dom/src/components/VoiceSearch.tsx

```diff
@@ -91,7 +95,7 @@ class VoiceSearch extends Component<VoiceSearchProps, VoiceListeningState> {
     this.setState(this.voiceSearchHelper.getState());
   }
 
-  public render() {
+  public render(): React.ReactNode {
```

Comment:
This is quite verbose and I've got the feeling this is gonna create some friction quickly.

Reply:
If everyone agree on this we could remove the rule from the ESLint config, WDYT?

Reply:
Updated to the new eslint-config-algolia release without this rule https://github.com/algolia/react-instantsearch/pull/2591/commits/2ab2fd3414eccfc1ba26e5c0a5329c3a256ddac6